font-patcher

```diff
@@ -817,6 +842,20 @@ class font_patcher:
             # does not overlap the bearings (edges)
             self.remove_glyph_neg_bearings(self.sourceFont[currentSourceFontGlyph])
 
+            # Check if the inserted glyph is scaled correctly for monospace
+            if self.args.single:
+                (xmin, _, xmax, _) = self.sourceFont[currentSourceFontGlyph].boundingBox()
+                if int(xmax - xmin) > self.font_dim['width'] * (1 + overlap):
+                    print("  Warning: Scaled glyph U+{:X} wider than one monospace width ({} / {} (overlap {}))".format(
```

Comment:
I think this should print on a new line. when progressbars are on it displays on the same line

Reply:
Changed.